resume-project/src/components/Contact.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const Contact = () => (
+  <section className="contact-info">
+    <h3>
+      <i className="fas fa-envelope"></i>
+      <a href="mailto:ajs476@nau.edu">ajs476@nau.edu</a>
+    </h3>
+    <h3>
```

Comment:
this structure:

```
<h3> 
  <i .....
   <a ....
</h3>
```

it's repeating a lot maybe split a different component and map through an array with the data to render it ?